test/debug_info/use_only_test.f90_mod.f90

```diff
@@ -0,0 +1,7 @@
+!! This file is compiled with other test use_only_test.f90
+!! Below run is just to satisfy test framework as run is must for 
+!! testcase to exist in this directory.
+!RUN: ls  %s
```

Comment:
The LLVM convention for this common idiom is:
```
!RUN: true
```

Reply:
Thanks. I am updating this,